app/src/main/java/com/eventyay/organizer/core/attendee/list/SwipeController.java

```diff
@@ -57,6 +65,11 @@ public int getMovementFlags(RecyclerView recyclerView, RecyclerView.ViewHolder v
     public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction) {
         int swipedPosition = viewHolder.getAdapterPosition();
         attendeesViewModel.toggleCheckInState(attendeeList, swipedPosition);
+
+        if (playSound) {
+            MediaPlayer mediaPlayer = MediaPlayer.create(context, R.raw.check_in_sound);
```

Comment:
I think raw fine not required. The media player can convert text to speech.

Reply:
Text to speech? Here, we are just playing a sound when a user is checked in.

Reply:
Yes, for that media player can speak the same from string also. It will reduce the app size.

Reply:
But how can the this type of sound be written as string? It doesn't contain words which we can convert using Text to Speech.

Reply:
Oh, I was thinking that it is speaking checked in.